src/LTaaS/DomainClient.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace UKFast\SDK\LTaaS;
+
+use UKFast\SDK\Page;
+use UKFast\SDK\LTaaS\Entities\Domain;
+
+class DomainClient extends Client
+{
+    protected $basePath = 'ltaas/';
+
+    /**
+     * Gets paginated response for all of the domains
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return int|Page
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getPage($page = 1, $perPage = 15, $filters = [])
+    {
+        $page = $this->paginatedRequest('v1/domains', $page, $perPage, $filters);
+        $page->serializeWith(function ($item) {
+            return new Domain($item);
+        });
+
+        return $page;
+    }
+
+    public function getDomain($domainId)
+    {
+        $response = $this->get('v1/domains/' . $domainId);
+
+        $body = $this->decodeJson($response->getBody()->getContents());
+
+        return new Domain($body->data);
+    }
+
+    /**
+     * Send the request to the API to store a new domain
+     * @param $name
+     * @param $verificationMethod
+     * @return mixed
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function store($name, $verificationMethod)
```

Comment:
please can you rename this method to `create` and require a Domain object be passed in populated with the required data. see other service clients for examples

Reply:
Changed to use the LTaaS Domain model

Reply:
didnt rename the method?

Reply:
Ah sorry will change that now